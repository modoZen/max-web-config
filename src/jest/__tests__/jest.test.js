const { getJestConfig } = require('../jest');

describe('Jest tests', () => {
  it('Should return prettier config values', () => {
    expect(getJestConfig()).toMatchSnapshot();
  });
});
