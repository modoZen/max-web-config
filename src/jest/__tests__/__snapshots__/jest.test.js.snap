// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest tests Should return prettier config values 1`] = `
{
  "collectCoverageFrom": [
    "**/src/**/!(index|main|*.test|*.stub|*.mock).js",
    "**/src/**/!(index|main|*.test|*.stub|*.mock).ts",
    "**/src/**/!(index|main|*.test|*.stub|*.mock).tsx",
    "!**/__mocks__/**",
    "!**/__stubs__/**",
    "!**/__tests__/**",
  ],
  "coverageThreshold": {
    "global": {
      "branches": 100,
      "functions": 100,
      "lines": 100,
      "statements": 100,
    },
  },
  "moduleNameMapper": {
    "^@/(.*)$": "<rootDir>/src/$1",
  },
  "resetMocks": true,
  "setupFiles": [
    "dotenv/config",
  ],
  "setupFilesAfterEnv": [
    "<rootDir>/jest.setup.ts",
  ],
  "testEnvironment": "jsdom",
  "transform": {
    "^.+\\.(js|ts|tsx)$": [
      "@swc/jest",
      {
        "jsc": {
          "transform": {
            "react": {
              "runtime": "automatic",
            },
          },
        },
      },
    ],
  },
}
`;
